src/py/flwr/client/supernode/app.py

```diff
@@ -255,13 +261,14 @@ def _parse_args_run_supernode() -> argparse.ArgumentParser:
     )
 
     parser.add_argument(
-        "client-app",
+        "app_dir",
         nargs="?",
-        default="",
-        help="For example: `client:app` or `project.package.module:wrapper.app`. "
-        "This is optional and serves as the default ClientApp to be loaded when "
-        "the ServerApp does not specify `fab_id` and `fab_version`. "
-        "If not provided, defaults to an empty string.",
+        default=None,
+        help="Specify the project directory to load and run the `ClientApp`. "
```

Comment:
```suggestion
        help="Specify the app directory to load and run the `ClientApp`. "
```

Reply:
should we update for consistency? do we need to indicate that this directory is the dir where `pyproject.toml` is (not where `clientapp.py` is defined)?

Reply:
Yeah, that makes a lot of sense. I will update the doc string to indicate that as per you suggest.